resolwe/flow/serializers/base.py

```diff
@@ -56,3 +59,10 @@ def __init__(self, instance=None, data=empty, **kwargs):
     def request(self):
         """Extract request object from serializer context."""
         return self.context.get('request', None)
+
+    def save(self):
```

Comment:
`def save(self, **kwargs):`

Reply:
Fixed in https://github.com/genialis/resolwe/pull/527/commits/b45d2f9abff4c82ab13a43515d89c99afad5c738